project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
-resolvers := Seq(DefaultMavenRepository, Resolver.jcenterRepo, Resolver.sonatypeRepo("releases"))
+resolvers += Resolver.jcenterRepo
```

Comment:
I think these were no longer being used.
@lolgab should be able to help with this one.

Reply:
The resolvers necessary to download all the dependencies are already included in sbt.
We only need `Resolver.jcenterRepo` because of `codacy-sbt-plugin`.
I don't know, however, if this requirement went away with `18.0.0`. Could you try removing it and see what's happening?

Reply:
Removing it results in an error.
`sbt.librarymanagement.ResolveException: Error downloading ohnosequences:ivy-s3-resolver:0.13.0`